### PR TITLE
Update component.yaml for dataflow and cmle with recent changes.

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/987)

### CODE REVIEW

File: components/gcp/dataflow/launch_python/component.yaml [hunk 1]
```diff
@@ -18,9 +18,14 @@ description: |
 inputs:
   - {name: python_file_path, description: 'The gcs or local path to the python file to run.'}
   - {name: project_id, description: 'The ID of the parent project.' }
```
Comment (gaoning777) [inline components/gcp/dataflow/launch_python/component.yaml]: We can probably use the types in this component yaml?

Comment (hongye-sun) [reply components/gcp/dataflow/launch_python/component.yaml]: Will do. I have a plan to update all component.yaml in one PR. Maybe I can just use this one.

Comment (gaoning777) [reply components/gcp/dataflow/launch_python/component.yaml]: I see. Feel free to do that all in one PR.

**Comment by gaoning777** [comment]
Thanks for adding the type information.

**Pull request merged**